### PR TITLE
fix docs typo

### PR DESCRIPTION
**Pull request by max8899**
Revelant -> Relelant

### CODE REVIEW

File: content/en/blog/_posts/2016-05-00-Coreosfest2016-Kubernetes-Community.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Don’t miss these great speaker sessions at the conference in **Berlin** :
 
 
 - [Kubernetes Performance & Scalability Deep-Dive](https://coreosfest2016.sched.org/event/6ckp/kubernetes-performance-scalability-deep-dive?iframe=no&w=i:100;&sidebar=yes&bg=no) by Filip Grzadkowski, Senior Software Engineer at Google
-- [Launching a complex application in a Kubernetes cloud](http://coreosfest2016.sched.org/event/6T0b/launching-a-complex-application-in-a-kubernetes-cloud) by Thomas Fricke and Jannis Rake-Revelant, Operations & Infrastructure Lead, immmr Gmbh (a service developed by the Deutsche Telekom’s R&D department)
+- [Launching a complex application in a Kubernetes cloud](http://coreosfest2016.sched.org/event/6T0b/launching-a-complex-application-in-a-kubernetes-cloud) by Thomas Fricke and Jannis Rake-Relelant, Operations & Infrastructure Lead, immmr Gmbh (a service developed by the Deutsche Telekom’s R&D department)
```
Comment (tengqm) [inline content/en/blog/_posts/2016-05-00-Coreosfest2016-Kubernetes-Community.md]: You don't want yourself to be renamed by others, right?

Comment (max8899) [reply content/en/blog/_posts/2016-05-00-Coreosfest2016-Kubernetes-Community.md]: yes, and you are very nice

**Pull request closed without merging**